nextflow_schema.json

```
@@ -105,6 +104,18 @@
                     "fa_icon": "fas fa-ban",
                     "hidden": true,
                     "help_text": "Do not load `igenomes.config` when running the pipeline. You may choose this option if you observe clashes between custom parameters and those supplied in `igenomes.config`."
+                },
+                "fasta_circular_target": {
+                    "type": "string",
+                    "description": "Specify the FASTA header of the target chromosome to extend (circularmapper only).",
+                    "help_text": "\n\nThe entry (chromosome, contig, etc.) in your FASTA reference that you'd like to be treated as circular.\n\nApplies only when providing a single FASTA file via `--fasta` (NOT multi-reference input - see reference TSV/CSV input).\n\n> Modifies tool parameter(s):\n> - circulargenerator `-s`\n",
```

```suggestion
                    "description": "Specify the FASTA header of the target chromosome to extend. Only applies when using `circularmapper`.",
                    "help_text": "The entry (chromosome, contig, etc.) in your FASTA reference that you'd like to be treated as circular.\n\nApplies only when providing a single FASTA file via `--fasta` (NOT multi-reference input - see reference TSV/CSV input).\n\n> Modifies tool parameter(s):\n> - circulargenerator `-s`\n",
```
I prefer actual sentences to the "notes prose" of the parenthesis system.